cmd/csaf_provider/create.go

```diff
@@ -87,20 +90,85 @@ func createFeedFolders(c *config, wellknown string) error {
 // and writes the CSAF field inside the file.
 func createSecurity(c *config, wellknown string) error {
```

Comment:
Maybe the function should be renamed, e.g. in `setupSecurity` :thinking:

Reply:
[x] done